packages/baggage-span-processor/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "@opentelemetry/baggage-span-processor",
+  "version": "0.1.0",
+  "description": "Baggage span processsor",
+  "main": "build/src/index.js",
```

Comment:
To match the format of other packages, we should add an `index.ts` file that exports the processor with `export { BaggageSpanProcessor } from './baggage-span-processor';`

Reply:
Added in https://github.com/open-telemetry/opentelemetry-js-contrib/pull/2114/commits/d3a73f796ae090875b5d1bc1ac667f5e94e970a5